### ISSUE TITLE
[DOC] add search to website

### ISSUE THREAD

**Issue by sozercan** [project-copacetic/copacetic#320]
### What kind of documentation improvement is needed?

Other

### What is the change that is needed?

website needs search integration setup through algolia

**Comment by bridgetkromhout** [comment]
https://docusaurus.io/docs/search#using-algolia-docsearch appears to have the details of where we would apply for the free tier of the Algolia indexing.

**Comment by sozercan** [comment]
@bridgetkromhout yes, that's the integration. are you interested in contributing?

**Comment by amanycodes** [comment]
I have applied for the free version of DocSearch.
Once I have received the api id and algolia index, i'll get started with the integration!
